packages/commonwealth/client/scripts/views/modals/ManageCommunityStakeModal/StakeExchangeForm/StakeExchangeForm.tsx

```diff
@@ -131,7 +135,7 @@ const StakeExchangeForm = ({
   const { trackAnalytics } = useBrowserAnalyticsTrack<BaseMixpanelPayload>({
     onAction: true,
   });
-
+  // eslint-disable-next-line @typescript-eslint/no-floating-promises
```

Comment:
is this really necessary?

Reply:
yes test are falling because of that even i didn't find anything wrong with code